pepper-apis/src/main/java/org/broadinstitute/ddp/model/files/FileUpload.java

```diff
@@ -125,9 +125,9 @@ public Instant getCreatedAt() {
     }
 
     /**
-     * Whether this file upload has been replaced by another upload. Replaced uploads should be removed.
+     * Returns timestamp of when file was finished uploading, null if not yet uploaded.
```

Comment:
Time when file was uploaded seems like a very useful piece of data to have. Adding it here so we don't have to lookup the bucket file whenever we need to know this.